include/NovelRT/Maths/GeoMatrix4x4F.h

```diff
@@ -22,139 +25,726 @@ namespace NovelRT::Maths
+         *      4 & 6 & 7 & 1
+         *      \end{bmatrix}
+         *      +
+         *      \begin{bmatrix}
+         *      4 & 7 & 2 & 5\\
+         *      5 & 1 & 8 & 4\\
+         *      8 & 3 & 2 & 2\\
+         *      6 & 7 & 6 & 8
+         *      \end{bmatrix}
+         *      =
+         *      \begin{bmatrix}
+         *      1+4 & 7+7 & 6+2 & 5+5\\
+         *      2+5 & 1+1 & 4+8 & 3+4\\
+         *      3+8 & 5+3 & 1+2 & 2+2\\
+         *      4+6 & 6+7 & 7+6 & 1+8
+         *      \end{bmatrix}
+         *      \rightarrow
+         *      \begin{bmatrix}
+         *      5 & 14 & 8 & 10\\
+         *      7 & 2 & 12 & 7\\
+         *      11 & 8 & 3 & 4\\
+         *      10 & 13 & 13 & 9
+         *      \end{bmatrix}
+         * \f]
+         * @param other The other matrix who's components get added together with this matrix.
+         * @return A matrix who's components consist of the entrywise sum of this and another GeoMatrix4x4F.
+         */
         inline GeoMatrix4x4F operator+(GeoMatrix4x4F other) const noexcept
         {
             return GeoMatrix4x4F(*reinterpret_cast<const glm::mat4*>(this) +
                                  *reinterpret_cast<const glm::mat4*>(&other));
         }
 
+        /**
+         * @brief Constructs a matrix containing the entrywise difference of this and another GeoMatrix4x4F.
+         *
+         * @details
+         * An example of a matrix that would be constructed from an entrywise difference: \f[
+         *      \begin{bmatrix}
+         *      1 & 7 & 6 & 5\\
+         *      2 & 1 & 4 & 3\\
+         *      3 & 5 & 1 & 2\\
+         *      4 & 6 & 7 & 1
+         *      \end{bmatrix}
+         *      -
+         *      \begin{bmatrix}
+         *      4 & 7 & 2 & 5\\
+         *      5 & 1 & 8 & 4\\
+         *      8 & 3 & 2 & 2\\
+         *      6 & 7 & 6 & 8
+         *      \end{bmatrix}
+         *      =
+         *      \begin{bmatrix}
+         *      1-4 & 7-7 & 6-2 & 5-5\\
+         *      2-5 & 1-1 & 4-8 & 3-4\\
+         *      3-8 & 5-3 & 1-2 & 2-2\\
+         *      4-6 & 6-7 & 7-6 & 1-8
+         *      \end{bmatrix}
+         *      \rightarrow
+         *      \begin{bmatrix}
+         *      -3 & 0 & 4 & 0\\
+         *      -3 & 0 & -4 & 1\\
+         *      -5 & 2 & -1 & 0\\
+         *      -2 & -1 & 13 & -7
+         *      \end{bmatrix}
+         * \f]
+         * @param other The other matrix who's components get subtracted from this matrix.
+         * @return A matrix who's components consist of the entrywise difference of this and another GeoMatrix4x4F.
+         */
         inline GeoMatrix4x4F operator-(GeoMatrix4x4F other) const noexcept
         {
             return GeoMatrix4x4F(*reinterpret_cast<const glm::mat4*>(this) -
                                  *reinterpret_cast<const glm::mat4*>(&other));
         }
 
+        /**
+         * @brief Constructs a matrix containing the product of this and another GeoMatrix4x4F.
```

Comment:
Yeah I'm really unsure if construct is the right term for mathematical operators.